tools/linter/clang_tidy/__main__.py

```diff
@@ -183,7 +183,8 @@ def main() -> None:
             f"Could not find '{options.clang_tidy_exe}'\n"
             + "We provide a custom build of clang-tidy that has additional checks.\n"
             + "You can install it by running:\n"
-            + "$ python3 tools/linter/install/clang_tidy.py"
+            + "$ python3 -m tools.linter.install.clang_tidy \n"
```

Comment:
Could you add a note in your PR description for why the original command failed? I looked at the logs but wasn't sure which snippet to look at.